lib/components/viewers/route-viewer.js

```diff
@@ -93,14 +99,13 @@ class RouteViewer extends Component {
           {agencySortedRoutes
             .map(route => {
               // Find operator based on agency_id (extracted from OTP route ID).
-              // TODO: re-implement multi-agency logos for route viewer.
-              // const operator = operatorForRoute(transitOperators, route) || {}
+              const operator = operatorForRoute(transitOperators, route) || {}
```

Comment:
If the operator is missing, the `<img>` tag is shown below, but it is a broken image. I think this default to empty object should be removed.